qiskit/providers/ibmq/api/rest/validation.py

```diff
@@ -63,3 +63,20 @@ class StatusResponseSchema(BaseSchema):
 
     # Required properties
     status = String(required=True, validate=OneOf([status.value for status in ApiJobStatus]))
+
+
+class BackendJobsLimitResponseSchema(BaseSchema):
+    """Schema for BackendJobsLimit"""
+
+    # required properties
+    maximum_jobs = Integer(required=True)
+    running_jobs = Integer(required=True)
```

Comment:
I think we should avoid making fields required if possible, so older versions of the provider won't blow up if the api changes the naming. You can instead of `missing=None` and add some wording saying `None` is returned if the information is not available.